include/odp/api/spec/packet_io.h

```diff
@@ -148,6 +148,17 @@ typedef enum odp_pktio_op_mode_t {
 
 } odp_pktio_op_mode_t;
 
+/**
+ * Packet input queue parameters override
+ */
+typedef struct odp_pktin_queue_param_ovr_t {
+	/** Thread group
+	  *
+	  * This parameter is considered only when queue type is
+	  * ODP_QUEUE_TYPE_SCHED. */
```

Comment:
It's better to document it as: "Override for schedule group in odp_schedule_param_t". For some reason  odp_schedule_param_t documentation label it thread group, but the type is odp_schedule_group_t so better use that term.

Reply:
OK